packages/inter-protocol/src/auction/offerBook.js

```diff
@@ -93,30 +144,54 @@ export const prepareScaledBidBook = baggage =>
         const { records } = this.state;
         return [...records.entries(M.gte(toBidScalingComparator(bidScaling)))];
       },
+      publishOffer(record) {
+        const key = toScaledRateOfferKey(record.bidScaling, record.seqNum);
+
+        return E(getBidDataRecorder(key)).write(
+          harden({
+            bidScaling: record.bidScaling,
+            wanted: record.wanted,
```

Comment:
the product spec calls this `maxBuy`. Should we

 - change it here (my preference)
 - translate the field name on the client
 - ask product if `wanted` is a good name?

cc @turadg